### ISSUE TITLE
Can you please explain why not do CV on LR?

### ISSUE THREAD

**Issue by liudragonfly** [emanuele/kaggle_pbr#3]
Can you please explain why not do CV on LR?
Is it possible to be overfit  without CV?

**Comment by emanuele** [comment]
CV is usually adopted for parameter estimation - e.g. setting parameters of classifiers - or for performance estimation - e.g. estimating generalization error. In blend.py the aim is to create the submission file for the competition, i.e. to compute the predictions for the final test set. There, the best you can do is to use the whole blended train set (`dataset_blend_train`) to fit the  LR and then to predict the final test set. If you use CV at that step - meaning that you split `dataset_blend_train` and, at each fold, you predict `dataset_blend_test`, then you would obtain multiple estimates of the final predictions, that you'd to need average in order to get the submission file. If you use CV, my guess is that the competition's score would be slightly inferior, given that, at each fold, you'd miss some valuable train examples. Anyway I may be wrong, so I invite you to try and let me know.
